client/src/components/payment/payment-modal.vue

```diff
@@ -92,10 +85,6 @@ export default class PaymentModal extends Mixins(
   /** The model value for the dialog component. */
   private isVisible = false
 
-  private get timerName () {
-    return this.$PAYMENT_COMPLETION_TIMER_NAME
```

Comment:
Can we get rid of all globals/properties that use this 'constant' also?

And also the configs?

Reply:
will do that

Reply:
And also PAYMENT_COMPLETION_TIMEOUT_MS I think.